pageserver/src/bin/pageserver.rs

```diff
@@ -77,6 +78,9 @@ fn main() -> anyhow::Result<()> {
         )
     })?;
 
+    let sentry_url = arg_matches.get_one::<String>("sentry-url");
```

Comment:
I'd call `get_one` right inside `init_sentry` arguments. I don't see it being used outside `init_sentry`

Reply:
I agree it's not used outside init_sentry, but I would still keep it as a separate variable for legibility purposes (otherwise the line gets is long and less legible).

Reply:
Let's not do that. `init_sentry` should only be responsible for one thing: actually initializing sentry regardless of argument names or parsing or whatever.